app/commands/command/batch/archive.rb

```diff
@@ -0,0 +1,16 @@
+module Command
+  module Batch
+    class Archive
+      def initialize(collection, ids)
+        @collection = collection
+        @ids = ids
+      end
+
+      def call
+        @collection
```

Comment:
Do we want to perform some extra checks before (batch)-archiving? Are there any cases when records (legislations in particular) should not have option to be archived? Those could be modeled using AM validations, similar as in `CsvDataUpload`

Reply:
I'm thinking that this will be safeguarded by the authorization rules, where not everyone will have access to all features. (I know, I need to finish that bit!). So for now, I wouldn't spend too much time thinking. As it will be brought in or handled from a different side.